docs/standard/containerized-lifecycle-architecture/key-takeaways/index.md

```diff
@@ -3,21 +3,21 @@ title: Key takeaways
 description: Review the key takeaways for the guide.
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 11/23/2018
+ms.date: 02/15/2019
 ---
 # Key takeaways
 
--   Container-based solutions provide important benefits of cost savings because containers are a solution to deployment problems caused by the lack of dependencies in production environments, therefore, improving DevOps and production operations significantly.
+- Container-based solutions provide important cost saving benefits because containers solve deployment problems caused by dependency failures in production environments, therefore, improving DevOps and production operations significantly.
```

Comment:
```suggestion
- Container-based solutions provide important cost-saving benefits because containers solve deployment problems caused by dependency failures in production environments, thereby improving DevOps and production operations significantly.
```

Reply:
So I finally understood the (slight) difference between therefore and thereby, thanks! 😊 (had to look it up anyway 😉)